tools/json-api-client/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@chainlink/json-api-client",
+  "private": true,
+  "version": "0.0.0",
+  "main": "./dist/src",
+  "types": "./dist/src",
```

Comment:
I don't think you need this field if they're in the same place. Could be wrong though

Reply:
This comes from https://github.com/smartcontractkit/chainlink/blob/develop/styleguide/package.json#L6

Reply:
Lets just keep it for consistency then